### PR TITLE
Update CommandBar.md to include VisibleBoundsPadding

### PR DESCRIPTION
**Pull request by MatFillion**
GitHub Issue (If applicable): #

## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
- [x] Contains **NO** breaking changes

### CODE REVIEW

File: doc/articles/controls/CommandBar.md [hunk 1]
```diff
@@ -437,15 +432,14 @@ Gets or sets a value indicating whether the user can interact with the control.
...
 # FAQ
 
-- > Why my content is underneath the CommandBar for iOS and the CommandBar underneath the StatusBar for Android ?
+- > Why my content is underneath the CommandBar for iOS and the CommandBar underneath the StatusBar/Notch for Android ?
 
-  You must set the `CommandBar` `Padding` to `StatusBarThickness` in order to fix those two known issues.
-  The `StatusBarThickness` resource is one which is programmatically added at runtime, with a value that changes depending on the platform/device.
+  You must use `VisibleBoundsPadding.PaddingMask="Top"` on `CommandBar` to properly support the notch or punch-holes on iOS and Android.
 
   ```xml
```
Comment (agneszitte-nventive) [inline doc/articles/controls/CommandBar.md]: Maybe precise `xmlns:toolkit="using:Uno.UI.Toolkit"` in the code sample also.

File: doc/articles/controls/CommandBar.md [hunk 2]
```diff
@@ -45,11 +45,7 @@ In this mode, the `CommandBar` can't be fully customized like other templatable
 
 ### Padding
 
-In this mode, you must set the `CommandBar` `Padding` to `StatusBarThickness` in order to fix two known issues :
-* On **iOS** the content will be underneath the `CommandBar`.
-* On **Android** the `CommandBar` will be underneath the `StatusBar`.
-
-(The `StatusBarThickness` resource is one which is programmatically added at runtime, with a value that changes depending on the platform/device.)
+You must use `VisibleBoundsPadding.PaddingMask="Top"` on `CommandBar` to properly support the notch or punch-holes on iOS and Android
```
Comment (davidjohnoliver) [inline doc/articles/controls/CommandBar.md]:
```suggestion
You must use `VisibleBoundsPadding.PaddingMask="Top"` on `CommandBar` to properly support the notch or punch-holes on iOS and Android.
```

File: doc/articles/controls/CommandBar.md [hunk 3]
```diff
@@ -437,21 +432,24 @@ Gets or sets a value indicating whether the user can interact with the control.
 
 # FAQ
 
-- > Why my content is underneath the CommandBar for iOS and the CommandBar underneath the StatusBar for Android ?
+- > Why my content is underneath the CommandBar for iOS and the CommandBar underneath the StatusBar/Notch for Android ?
```
Comment (davidjohnoliver) [inline doc/articles/controls/CommandBar.md]:
```suggestion
- > Why is my content underneath the `CommandBar` for iOS and the `CommandBar` underneath the StatusBar/Notch for Android ?
```

**Comment by ghuntley** [comment]
Thanks so much for dropping on by to improve Uno @MatFillion (:D). Can you please drop by https://platform.uno/uno-is-joining-hacktoberfest-2019/ and follow the instructions on that page and we'll send you some swag.

**Pull request merged**